hail/python/hail/methods/qc.py

```diff
@@ -567,6 +695,32 @@ def vep(dataset: Union[Table, MatrixTable], config=None, block_size=1000, name='
      - ``GRCh37``: ``gs://hail-us-vep/vep85-loftee-gcloud.json``
      - ``GRCh38``: ``gs://hail-us-vep/vep95-GRCh38-loftee-gcloud.json``
 
+    The config file when using the service backend must contain the following fields:
+
+     - `command` (array of string) -- The command line to run for a VEP job for a partition.
+     - `csq_header_command` (array of string) -- The command line to run when generating the consequence header.
+     - `env` (object) -- A map of environment variables to values to add to the environment when invoking the command.  The value of each object member must be a string.
+     - `vep_json_schema` (string): The type of the VEP JSON schema (as produced by the VEP when invoked with the `--json` option).  Note: This is the old-style 'parseable' Hail type syntax.  This will change.
+     - `image` (string) -- The docker image to run VEP.
+     - `data_bucket` (string) -- The location where the VEP data is stored.
+     - `data_mount` (string) -- The location in the container where the data should be mounted.
+     - `region` (string) -- The cloud region in which to run VEP jobs.
```

Comment:
I think I'm totally convinced now that this should not be a URL to some remote config file. It should just be a dictionary (or maybe a bunch of arguments?). We never read this file in a Batch worker, AFAICT, it's just read here on the client-side.